com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1187,47 +1153,31 @@ internal static FbxVector4 ToFbxVector4(FbxDouble3 v)
             return new FbxVector4(v.X, v.Y, v.Z);
         }
 
-        internal static FbxDouble3 ConvertToRightHandedEuler(Vector3 rot)
-        {
-            rot.y *= -1;
-            rot.z *= -1;
-            return ToFbxDouble3(rot);
-        }
-
         /// <summary>
         /// Euler to quaternion without axis conversion.
```

Comment:
Let's update the comment: Euler roll/pitch/yaw (ZXY rotation order) to quaternion.